x/auction/keeper/querier_test.go

```diff
@@ -107,7 +107,9 @@ func (suite *QuerierTestSuite) TestQueryAuctions() {
 	// Set up request query
 	query := abci.RequestQuery{
 		Path: strings.Join([]string{custom, types.QuerierRoute, types.QueryGetAuctions}, "/"),
-		Data: types.ModuleCdc.MustMarshalJSON(types.NewQueryAllAuctionParams(1, TestAuctionCount)),
+		Data: types.ModuleCdc.MustMarshalJSON(
```

Comment:
Add issue to update tests to include tests with flags - low priority.

Reply:
https://github.com/Kava-Labs/kava/issues/526